stdlib/crypto/src/main/ballerina/crypto/natives.bal

```diff
@@ -40,10 +40,11 @@ documentation {
 
     P{{baseString}} The string to be hashed
     P{{keyString}} The key string
+    P{{keyEncoding}} The encoding of the key
     P{{algorithm}} The hashing algorithm to be used
     R{{}} The hashed string
 }
-public extern function hmac(string baseString, string keyString, Algorithm algorithm) returns (string);
+public extern function hmac(string baseString, string keyString, string keyEncoding = "UTF-8", Algorithm algorithm) returns (string);
```

Comment:
Can we use a type to describe keyEncoding to be used?

Reply:
If we use a type we won't be able to add a default value, therefore I have kept it as a string.

Reply:
is it keyEncoding or encoding scheme to be used by the function/algorithm?

Reply:
It is the encoding of the key.